src/Tizen.NET.MaterialComponents/Components/MMenu.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using ElmSharp;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public class MMenu : ContextPopup, IColorSchemeComponent
+    {
+        IList<MMenuItem> _items = new List<MMenuItem>();
```

Comment:
could you explain why we create a list for menuitem? Couldn't we reuse `ContextPopup`'s one?

Reply:
It required to provide a list of items because context_popup does not provide a list of items.
context_popup is provide only `Append` API.